conf/messages.en

```diff
@@ -38,9 +38,28 @@ error.number = Please enter a valid number
 error.required = Please enter a value
 error.summary.title = There is a problem
 
-index.title = trader-goods-profiles-frontend
-index.heading = trader-goods-profiles-frontend
-index.guidance = Welcome to your new frontend. Please see the README file for a guide to getting started.
+profileSetup.title = Setting up your profile
+profileSetup.h1 = Setting up your profile
```

Comment:
Why are both of these the same - can we combine them?

Reply:
Not sure if the title and h1 will be same in the new versions. Needs to be confirmed may be

Reply:
I think we can check in the prototype when you inspect the html

Reply:
But I agree with you so it's resolved :)